fiftyone/core/stages.py

```diff
@@ -3039,8 +3045,9 @@ def validate(self, sample_collection):
             # Assume the user directly specified the subfield to use
             self._location_key = self._location_field
 
-        # These operations require a spherical index
-        sample_collection.create_index([(self._location_key, "2dsphere")])
+        if self._create_index:
+            # These operations require a spherical index
```

Comment:
Should we add a logging statement here for creating index?

Reply:
Definitely a reasonable suggestion. This module doesn't have a convention of logging things so I'll punt on it now for consistency